### PR TITLE
Standardize state voter refs

### LINKED ISSUE #149: Standardize state voter ref
To:

CO{12345}
FL{12345}
OK{12345}

etc.

And add some tests to test_transformers.py to cover this.

### PR DESCRIPTION
**Pull request by JakeStevens**
# What's in this PR?

Adds check to test_transformer for state voter refs in the form <STATE_ID><DIGITS>
Changes some states to follow this convention.

## References
Fixes #149

### CODE REVIEW

**Comment by JakeStevens** [comment]
Hmm, not sure how to remove the commits from my previous update..

File: src/python/national_voter_file/us_states/co/transformer.py [hunk 1]
```diff
@@ -162,6 +161,11 @@ def extract_mailing_address(self, input_dict):
 
     #### Political methods #####################################################
 
+    def extract_state_voter_ref(self, input_dict):
+        return {'STATE_VOTER_REF' : 'CO' + input_dict['VOTER_ID']}
+
+    extract_county_voter_ref = lambda self, i: {'COUNTY_VOTER_REF': None}
```
Comment (slavingia) [inline src/python/national_voter_file/us_states/co/transformer.py]: this shouldn't be needed as it's in the col_map above.

File: src/python/national_voter_file/tests/test_transformers.py [hunk 2]
```diff
@@ -17,6 +17,11 @@
 
 TEST_STATES = ['co', 'de', 'fl', 'mi', 'nc', 'ny', 'oh', 'ok', 'ut', 'wa']
 
+STATE_CODE_MAP = {'Colorado' : 'CO', 'Delaware' : 'DE', 'Florida' : 'FL',
```
Comment (slavingia) [inline src/python/national_voter_file/tests/test_transformers.py]: we can probably just have one dictionary, call it TEST_STATES, and then use TEST_STATES.values to get a list of the state initials? if not that python method, there's another one :)

**Comment by slavingia** [comment]
those rogue changes should go away when once squash+merge happens

**Pull request merged**